jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -319,6 +319,40 @@ def parameterized(harness_group: Iterable[Harness],
   ]
 )
 
+lax_top_k = tuple( # random testing
+  Harness(f"_inshape={jtu.format_shape_dtype_string(shape, dtype)}_k={k}",
+          lax.top_k,
+          [RandArg(shape, dtype), StaticArg(k)],
+          shape=shape,
+          dtype=dtype,
+          k=k)
+  for dtype in jtu.dtypes.all #[np.float32, np.int32, np.uint32]
+  for shape in [(3,), (5, 3)]
+  for k in [-1, 1, 3, 4]
+  for rng_factory in [jtu.rand_default]
+) + tuple( # stability test
+  Harness(f"stability_inshape={jtu.format_shape_dtype_string(arr.shape, arr.dtype)}_k={k}",
```

Comment:
I am curious how is stability involved in top_k? Is it about testing cases when we have repeated values?

Reply:
`top_k` returns the top k elements as well as their indices within the array; for that reason, it is important to know how the case in which two elements are the same is treated. The [TensorFlow docs](https://www.tensorflow.org/api_docs/python/tf/math/top_k) mentions that "If two elements are equal, the lower-index element appears first.", hence the stability check.